src/atomate2/vasp/flows/hiphive.py

```diff
@@ -389,9 +389,11 @@ def make(
             # supercell_matrix_kwargs=supercell_matrix_kwargs,
             supercell_matrix=supercell_matrix,
             # rattled_structures=vasp_static_calcs.output,
-            rattled_structures=vasp_displacement_calcs.output.structure,
+            # rattled_structures=vasp_displacement_calcs.output.structure,
+            rattled_structures=vasp_displacement_calcs.output,
```

Comment:
Why define the same variable (`vasp_displacement_calcs.output`) with two names (`rattled_structures` and `forces` )? It is obviously not right.